friprosveta/views.py

```diff
@@ -478,6 +479,46 @@ def allocations_json(request, timetable_slug=None):
     filtered_allocations = _allocation_set(param_ids,
                                            tt.allocations,
                                            request.user.is_staff)
+
+    #returns more information about allocations
+    r = request.GET
+    if 'mode' in r:
+        l = r['mode']
+        if l == 'ext':
+            AllocationVM = namedtuple('AllocationVM', ['object', 'day_index', 'hour_index', 'duration'])
+            weekday_mapping = {wd[0]: i for i, wd in enumerate(WEEKDAYS)}
+            hour_mapping = {wh[0]: i for i, wh in enumerate(WORKHOURS)}
+            allocation_vms = [AllocationVM(
+                object=a,
```

Comment:
Please use descriptive variable names, such as `allocation` instead of `a`.

Reply:
Changed variable names.